package.json

```diff
@@ -56,18 +58,28 @@
     "@types/react-dom": "^16.9.3",
     "awesome-typescript-loader": "^5.2.1",
     "axios": "^0.18.1",
+    "babel-eslint": "^10.0.3",
```

Comment:
Why do we need this?

Reply:
It's a peer dependency of [@cognite/eslint-config](https://github.com/cognitedata/eslint-config-cognite)